### PR TITLE
I'm hunting rabbits

### PR DESCRIPTION
**Pull request by jasonmotha**
Various bugs found with the pre-pro workflow.

### CODE REVIEW

**Comment by jasonmotha** [comment]
Just putting this through as is. I might create another PR later.

**Review by shot0829 — APPROVED** [review]

**Comment by shot0829** [comment]
its still draft? o.o

File: templates/gmsim/20.4.1.2/root_defaults.yaml [hunk 1]
```diff
@@ -23,6 +23,12 @@ ims:
     - rotd50
     - rotd100_50
   pSA_periods: [0.01, 0.02, 0.03, 0.04, 0.05, 0.075, 0.1, 0.12, 0.15, 0.17, 0.2, 0.25, 0.3, 0.4, 0.5, 0.6, 0.7, 0.75, 0.8, 0.9, 1.0, 1.25, 1.5, 2.0, 2.5, 3.0, 4.0, 5.0, 6.0, 7.5, 10.0]
+VM:
+  hh: 0.2
+  VM_Version: 2.06
+  VM_Topo: BULLDOZED
+  PGV_Threshold: 2
+  Ds_Multiplier: 0.75
```
Comment (jpa198) [inline templates/gmsim/20.4.1.2/root_defaults.yaml:31]: This opens bit of a can of worms for the other gmsim version files

**Review by jpa198 — APPROVED** [review]

File: templates/gmsim/20.4.1.2/root_defaults.yaml (hunk 1 above)
Comment (jasonmotha) [reply templates/gmsim/20.4.1.2/root_defaults.yaml:31]: All of the 20.4 series version files now have this VM section in it.

**Pull request merged**